Rakefile

```diff
@@ -1,22 +1,22 @@
 require 'rake'
 require 'rake/testtask'
 
-require_relative 'lib/tasks/exercise_test_task'
+require_relative 'lib/tasks/exercise_tests'
 
 task default: 'test'
 
-desc 'Run all development and exercise tests'
+desc 'Run individual exercises or run all development and exercise tests'
 task :test do
   exercises = ARGV.take_while { |e| e != '--' }.drop(1)
 
   if exercises.any?
-    exercises.each do |e|
-      task(e) { Rake::Task['test:exercise'].execute(exercise: e) }
-    end
-  else
-    Rake::Task['test:dev'].invoke
-    Rake::Task['test:exercises'].invoke
+    exercises.each { |e| task(e) { } }
```

Comment:
This defining no-op tasks for the given exercises so that rake doesn't do this:

```
rake aborted!
Don't know how to build task 'all-your-base'
```